src/plugins/data_views/common/data_views/data_views.test.ts

```diff
@@ -844,7 +855,7 @@ describe('IndexPatterns', () => {
 
     test('dont set defaultIndex without capability allowing advancedSettings save', async () => {
```

Comment:
Is this description still accurate for this test if we no longer rely on capabilities API?

Reply:
The advanced settings and therefore the default data view rely on the capabilites API, therefore the test is still relevant.